cfme/common/__init__.py

```diff
@@ -286,10 +286,12 @@ def get_tags(self, tenant="My Company Tags"):
             List of tags in format "Tag_category: Tag_name"
         """
         view = navigate_to(self, 'Details')
-        tags = view.tag.read()
+        tags = view.smart_management.read()
+        if isinstance(tags, dict):
+            tags = tags[tenant]
         if tags == 'No {} have been assigned'.format(tenant):
             return []
-        return filter(None, re.split(r'(.*?):\s*\S+\s?', tags))
```

Comment:
It was capital ```S``` earlier, giving only ```[' ', 'Department', ' ']```,
After changing it to ```s```, giving ```[' ', 'Department', 'Support']``` which can be filter-out to obtain list !